.github/workflows/application.yml

```diff
@@ -113,6 +113,15 @@ jobs:
                 name: Run PHPStan
                 run: vendor/bin/phpstan analyse
                 if: always() && steps.end-of-setup.outcome == 'success'
+            
+            -
+                name: Run ComposerRequireChecker
+                run: |
+                    (cat composer.json | jq '.["autoload-dev"]["psr-4"] |= . + {"Sylius\\Behat\\": "src/Sylius/Behat/"}' | jq 'del(.autoload["psr-4"]["Sylius\\Behat\\"])') > _composer.json
```

Comment:
Can you explain this?

Reply:
The `Sylius\Behat` namespace is in the `autoload` section. For ComposerRequireCheker this means that all symbols used in those classes must be in the `require` section of the `composer.json` and not in the `require-dev` one. Unless we really want to add all Behat, Mink and other packages to the `require` section we have to do that.

This is a workaround. IMHO the real solution is to move the whole Behat contexts, pages and services classes to a separate package that can be installed by those who needs them in their own Behat test suites. But obviously this is not in the scope of this PR.

Reply:
Thanks!